ludwig/schema/decoders/base.py

```diff
@@ -266,3 +266,51 @@ def module_name(cls):
         description="Initializer for the bias vector.",
         parameter_metadata=DECODER_METADATA["Classifier"]["bias_initializer"],
     )
+
+
+@DeveloperAPI
+@register_decoder_config("tart", [BINARY, CATEGORY], model_types=[MODEL_ECD, MODEL_GBM, MODEL_LLM])
```

Comment:
not 100% sure of the implications of making a tart decoder available for gbm and llm. as long as we get outputs from them in the same shape as the ones expected from ecd, that's probably ok, but I will make sure of that first.